src/Blackboard/Provider.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace SocialiteProviders\Blackboard;
+
+use GuzzleHttp\RequestOptions;
+use Illuminate\Support\Arr;
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
+{
+    /**
+     * Unique Provider Identifier.
+     */
+    public const IDENTIFIER = 'BLACKBOARD';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopes = ['read'];
+
+    /**
+     * {@inheritdoc}
+     */
+    protected $scopeSeparator = ' ';
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function additionalConfigKeys()
+    {
+        return ['subdomain'];
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getAccessTokenResponse($code)
+    {
+        $basic_auth = base64_encode($this->clientId.':'.$this->clientSecret);
```

Comment:
Can we use the guzzle basic auth param here?
https://docs.guzzlephp.org/en/stable/request-options.html#auth